nubis/puppet/sites.pp

```diff
@@ -37,6 +37,38 @@
   ],
 }
 
+nubis::static { 'local-start':
+  servername    => 'en-us.start.mozilla.org',
```

Comment:
This servername seems wrong, but I assume it's copied forward _correctly_ from the IT puppet code preceding this. Assuming that, OK.

Reply:
You are correct. I copied this from the Apache config preceding this.